models/fivetran_platform__usage_mar_destination_history.sql

```diff
@@ -10,12 +10,24 @@ credits_used as (
     from {{ ref('stg_fivetran_platform__credits_used') }}
 ),
 
-useage_cost as (
+usage_cost as (
 
     select *
     from {{ ref('stg_fivetran_platform__usage_cost') }}
 ),
 
+transformation_runs as (
+
+    select
+        destination_id,
+        measured_month,
+        sum(case when free_type = 'PAID' then model_runs else 0 end) as paid_model_runs,
+        sum(case when free_type != 'PAID' then model_runs else 0 end) as free_model_runs,
```

Comment:
We've seen cases in the past where the casing of strings in the Fivetran Platform data model could possibly change in the future. Can we make this more future proof and simple to a lower casing search?
```suggestion
        sum(case when lower(free_type) = 'paid' then model_runs else 0 end) as paid_model_runs,
        sum(case when lower(free_type) != 'paid' then model_runs else 0 end) as free_model_runs,
```

Reply:
Hey I saw this was added but don't we do `upper(free_type) as free_type` in the staging model?

Reply:
Oh you are 100% correct thanks for noticing that. I think it makes more sense to keep casing in staging, but I just realized everywhere else we do a lower albet in the transforms. I'll do a `lower` in the staging and remove it from the transforms. 

cc @fivetran-joemarkiewicz